index.html

```diff
@@ -1724,6 +1724,78 @@ <h4>CredentialRegistry</h4>
         </pre>
       </section>
 
+      <section>
+        <h4>OpenID4 Verifiable Credential Issuance</h4>
+        <p>The <code>OID4VCI</code> service allows publication of the <a href="https://openid.net/openid4vc/">OpenID for Verifiable Credential Issuance (OID4VCI)</a> domain, where the <code>.well-known</code> files (<code>oauth-authorization-server</code> & <code>openid-credential-issuer</code>) can be found. These files MUST contain the endpoints and meta data for <a href="https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html">OpenID for Verifiable Credential Issuance (OpenID4VCI)</a>.</p>
+        <table class="simple" style="width: 100%;">
+          <thead>
+            <tr>
+              <th>Normative Definition</th>
+              <th>JSON-LD</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>
+                <a href="https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html">OpenID4VCI</a>
+              </td>
+              <td>
+                <a href="https://ssi.eecc.de/api/registry/context/openid4vc">OpenID4VCI</a>
+              </td>
+            </tr>
+          </tbody>
+        </table>
+
+        <pre class="example" title="Example of service and serviceEndpoint properties">
+{
+  ...
+  "service": [
+    {
+      "id": "did:example:123#oid4vci",
+      "type": "OID4VCI",
+      "serviceEndpoint": "https://did.example.com/"
+    }
+  ]
+}
+        </pre>
+      </section>
+
+      <section>
+        <h4>OpenID4 Verifiable Presentation</h4>
+        <p>The <code>OID4VP</code> service allows publication of the <a href="https://openid.net/openid4vc/">OpenID for Verifiable Presentation (OID4VP)</a> authorization endpoint, which is the entry point for the <a href="https://openid.net/specs/openid-4-verifiable-presentations-1_0.html">presentation flow</a>.</p>
+        <table class="simple" style="width: 100%;">
+          <thead>
+            <tr>
+              <th>Normative Definition</th>
+              <th>JSON-LD</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>
+                <a href="https://openid.net/specs/openid-4-verifiable-presentations-1_0.html#name-authorization-request">OpenID4VP</a>
+              </td>
+              <td>
+                <a href="https://ssi.eecc.de/api/registry/context/openid4vc">OpenID4VP</a>
+              </td>
+            </tr>
+          </tbody>
+        </table>
+
+        <pre class="example" title="Example of service and serviceEndpoint properties">
+{
+  ...
+  "service": [
+    {
+      "id": "did:example:123#oid4vp",
+      "type": "OID4VP",
+      "serviceEndpoint": "https://did.example.com/authorize"
+    }
+  ]
+}
```

Comment:
In general the Wallet Metadata from the OpenID4VP spec but technically everything that is described in RFC8414 which includes `authorization_endpoint`.

Reply:
Is there a specific advantage to offer both URL and the Object? From my point of view the use of OpenID4VC prerequisites a domain which hosts the .well-known files, what makes it a bit redundant to port them into the DIDDocument.

Reply:
I guess you are right, it should be OAuth2 issuer identifier only to avoid confusion. I was just thinking to support Wallets that don't have a backend but hosting a static file is even possible for wallets without a backend.

Reply:
I would keep it simple for now and just support:
- OAuth2 issuer identifier for OpenID4VP 
- Credential issuer identifier for OpenID4VCI

Both should be used for the `id` property in the service endpoint. If one needs the metadata, the service endpoint spec can be updated in the future.

Reply:
Integrated in https://github.com/w3c/did-spec-registries/pull/513/commits/e7c2827d78b79bbb4af97036673d4df18b116518